tests/driver.sh

```diff
@@ -323,4 +323,20 @@ int main()
 }
 EOF
 
+# #ifdef...#else...#endif
+try_ 100 << EOF
+#define A 100
+#define B 200
+int main()
+{
+    int x;
+#ifdef A
+    x = A;
+#else
+    x = B;
+#endif
+    return x;
```

Comment:
The return value is crucial for automated tests. Ensure that 0 is returned in the expected path.